server/core/src/storage.ts

```diff
@@ -333,6 +333,25 @@ class TServerStorage implements ServerStorage {
     return result
   }
 
+  async processMove (ctx: MeasureContext, tx: Tx): Promise<Tx[]> {
+    const actualTx = this.extractTx(tx)
+    if (!this.hierarchy.isDerived(actualTx._class, core.class.TxUpdateDoc)) return []
+    const rtx = actualTx as TxUpdateDoc<Doc>
+    if (rtx.operations.space === undefined || rtx.operations.space === rtx.objectSpace) return []
+    const result: Tx[] = []
+    const factory = new TxFactory(core.account.System)
+    for (const [, attribute] of this.hierarchy.getAllAttributes(rtx.objectClass)) {
+      if (!this.hierarchy.isDerived(attribute.type._class, core.class.Collection)) continue
+      const collection = attribute.type as Collection<AttachedDoc>
+      const allAttached = await this.findAll(ctx, collection.of, { attachedTo: rtx.objectId, space: rtx.objectSpace })
+      const allTx = allAttached.map(({ _class, space, _id }) =>
+        factory.createTxUpdateDoc(_class, space, _id, { space: rtx.operations.space }, rtx.retrieve)
```

Comment:
You need to ignore attached with space not same as previous doc value.

Reply:
Space is specified in `findAll`, can it return attached with different space?